client/python/README.md

```diff
@@ -3,22 +3,22 @@
 To install from source, run:
 
 ```bash
-$ python setup.py install
```

Comment:
As per https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html this is not supported anymore.